### PR TITLE
WIP: Page menu component

### PR DESCRIPTION
**Pull request by fmubaidien**
Created a pagebutton component and a page menu component, added pages to be linked to through clicking on the menu

### CODE REVIEW

**Comment by geokhoury** [comment]
#### 📝Changed routes:


| [**/FAQ**](https://start-mwjju5pw1-josa.vercel.app/FAQ) | [**/guides**](https://start-mwjju5pw1-josa.vercel.app/guides) |
|:-:|:-:|
|<a href="https://start-mwjju5pw1-josa.vercel.app/FAQ">Screenshot of /FAQ</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/xCeZ5rSUDhapA3E4mRbHe0rke-SmhaDdLty-6NAVqyR302P9MtmBHklGHJJH330XGB6gFpOl6SxldLTPJrmHqg.png">(view full size)</a> |<a href="https://start-mwjju5pw1-josa.vercel.app/guides">Screenshot of /guides</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/b1L5w2uohh_KM8Egp7pSs4jL0_Bub1du7-F35FpLcjcHai8XhGJSfL2W8-BufYhSVUuhWK2wYSCYmd2m-g6big.png">(view full size)</a> |



| [**/knowledgebase**](https://start-mwjju5pw1-josa.vercel.app/knowledgebase) | [**/questions**](https://start-mwjju5pw1-josa.vercel.app/questions) |
|:-:|:-:|
|<a href="https://start-mwjju5pw1-josa.vercel.app/knowledgebase">Screenshot of /knowledgebase</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/NABEy3ulbhsWtCfC603BDkBLFrOKC0Jd4injbgaTYNC2hGikod5iOJhUkVrETSa0M6_dUfbNze-JWbRODhfFHA.png">(view full size)</a> |<a href="https://start-mwjju5pw1-josa.vercel.app/questions">Screenshot of /questions</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/sqcOS5uApVFPywaoaJ1tp3jN5d33e54zlDPdaXX1nFcF3IO7V6pGdtS-EyrwkXT7vsAGpT_VIV1Tl9gZFxz0UQ.png">(view full size)</a> |



| [**/quicklinks**](https://start-mwjju5pw1-josa.vercel.app/quicklinks) | [**/services**](https://start-mwjju5pw1-josa.vercel.app/services) |
|:-:|:-:|
|<a href="https://start-mwjju5pw1-josa.vercel.app/quicklinks">Screenshot of /quicklinks</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/_2AbzusA0fale49yKFayUDUT60Z0iT32vpsP7IfWlc9T72Hxk5eOoLv06zX5FKN73hEpndh8ETQyfyKMJB7ewA.png">(view full size)</a> |<a href="https://start-mwjju5pw1-josa.vercel.app/services">Screenshot of /services</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/gf7uXSx-rdM_KNJ1L2gCFcVQdz6clJjVz_BwEkOauv9H2ThqwuPchtg4UQY3fxpL3boBXTz6fB-3PKifFAXgfA.png">(view full size)</a> |







Commit 6ec595f59c9921ae7995902d59677484e1c5fd61 (https://start-mwjju5pw1-josa.vercel.app).

File: components/PageMenu.vue [hunk 1]
```diff
@@ -0,0 +1,44 @@
+<template>
+  <div>
+    <h3 class="heading">Pages</h3>
```
Comment (wissamataleh) [inline components/PageMenu.vue:3]: * Remove class definitions for styling, and add styling for headers in typography css

File: components/PageMenu.vue [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+      knowledgebase: 'knowledgebase',
+      quickLinks: 'quickLinks',
+      FAQ: 'FAQ'
+    }
+  }
+}
+</script>
+
+<style>
+  ul {
```
Comment (wissamataleh) [inline components/PageMenu.vue:30]: Use classes to style menu

File: components/PageMenu.vue [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+      services: 'services',
+      knowledgebase: 'knowledgebase',
+      quickLinks: 'quickLinks',
+      FAQ: 'FAQ'
+    }
+  }
+}
+</script>
+
+<style>
```
Comment (wissamataleh) [inline components/PageMenu.vue:29]: Use scoped style

**Review by wissamataleh — CHANGES_REQUESTED** [review]
1- Review comments on files.
2- Delete all test pages.
3- Revert deleting `package-lock.json`

**Comment by wissamataleh** [comment]
- Use hyphens in classes instead of camel case

**Pull request closed without merging**